src/pretix/plugins/paypal/payment.py

```diff
@@ -56,24 +68,77 @@ def settings_form_fields(self):
                      label=_('Secret'),
                      max_length=80,
                      min_length=80,
-                 ))
+                 )),
+            ]
+
+        d = OrderedDict(
+            fields + [
+                ('endpoint',
+                 forms.ChoiceField(
+                     label=_('Endpoint'),
+                     initial='live',
+                     choices=(
+                         ('live', 'Live'),
+                         ('sandbox', 'Sandbox'),
+                     ),
+                 )),
             ] + list(super().settings_form_fields.items())
         )
+
         d.move_to_end('_enabled', False)
         return d
 
+    def get_connect_url(self, request):
+        request.session['payment_paypal_oauth_event'] = request.event.pk
+
+        self.init_api()
+        return Tokeninfo.authorize_url({'scope': 'openid profile email'})
+
     def settings_content_render(self, request):
-        return "<div class='alert alert-info'>%s<br /><code>%s</code></div>" % (
-            _('Please configure a PayPal Webhook to the following endpoint in order to automatically cancel orders '
-              'when payments are refunded externally.'),
-            build_global_uri('plugins:paypal:webhook')
-        )
+        if self.settings.connect_client_id and not self.settings.secret_key:
+            # Use PayPal connect
+            if not self.settings.connect_user_id:
+                return (
+                    "<p>{}</p>"
+                    "<a href='{}' class='btn btn-primary btn-lg'>{}</a>"
+                ).format(
+                    _('To accept payments via PayPal, you will need an account at PayPal. By clicking on the '
+                      'following button, you can either create a new PayPal account connect pretix to an existing '
+                      'one.'),
+                    self.get_connect_url(request),
+                    _('Connect with <i class="fa fa-paypal"></i> PayPal')
```

Comment:
Can we refactor this into
```
 _('Connect with {icon} PayPal').format(icon='<i class="fa fa-paypal"></i>')
```
to avoid confusion with translators?

Reply:
Yes, definitely. :-)